src/shared/json-queue.c

```diff
@@ -9,6 +9,35 @@
 
 #include "shared.h"
 
+#ifdef UNIT_TESTING
+    #define static
+#endif
+static void file_sleep(void);
+static int Handle_JQueue(file_queue *fileq, int flags) __attribute__((nonnull));
+
+/* To translate between month (int) to month (char) */
+static const char *(s_month[]) = {
+    "Jan", "Feb", "Mar", "Apr", "May", "Jun",
+    "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"
+};
+
+static void file_sleep() {
+#ifndef WIN32
+    struct timeval fp_timeout;
+
+    fp_timeout.tv_sec = FQ_TIMEOUT;
+    fp_timeout.tv_usec = 0;
+
+    /* Wait for the select timeout */
+    select(0, NULL, NULL, NULL, &fp_timeout);
+
+#else
```

Comment:
The report daemon only works in managers, so this `Sleep()` will never run.

Reply:
I removed this section of the code here and in `file-queue.c` file.
https://github.com/wazuh/wazuh/pull/4256/files#diff-399a48392f0369ec1b77cbfb563939dcR25